ops/create-cluster.sh

```diff
@@ -55,7 +55,7 @@ done
 readonly graphql_service_host="fortis-services.${DEIS_ROUTER_HOST_ROOT}.nip.io"
 readonly fortis_interface_host="fortis-interface.${DEIS_ROUTER_HOST_ROOT}.nip.io"
 readonly feature_service_db_conn_str="${FEATURE_SERVICE_DB_CONNECTION_STRING}"
-readonly feature_service_host="feature-service.${DEIS_ROUTER_HOST_ROOT}.nip.io"
+readonly feature_service_host="http://feature-service.${DEIS_ROUTER_HOST_ROOT}.nip.io"
```

Comment:
No, I just was referring to the name. (I don't think hosts typically have `http://` in front that's all.